hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: patch
+
+This improves the shrinker. It can now reorder examples: 3 1 2 becomes 1 2 3.
```

Comment:
nit: Maybe this would read better with Python values. e.g. "e.g. by shrinking `[3, 1, 2]` to `[1, 2, 3]`"?

Reply:
I'll leave it as-is. I originally wrote it that way, but changed it before PRing it, because it's byte arrays and that syntax suggests lists. Both aren't optimal. :)